File: Python/1134 - Type of Fuel.py
alcohol=0
gasoline=0
diesel=0
x=0
while x!=4:
    x=int(input())
    if x==4:
        break
    elif x==1:
        alcohol+=1
    elif x==2:
        gasoline+=1
    elif x==3:
        diesel+=1
print("MUITO OBRIGADO")
print(f"Alcool: {alcohol}")
print(f"Gasolina: {gasoline}")
print(f"Diesel: {diesel}")

File: Java/1134 - Type of Fuel.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int combustivel = 0, alcool = 0, gasolina = 0, diesel = 0;

        while (combustivel != 4) {
            combustivel = sc.nextInt();
            if (!(combustivel > 4 || combustivel < 1)) {
                if (combustivel == 1) {
                    alcool++;
                } else if (combustivel == 2) {
                    gasolina++;
                } else if (combustivel == 3) {
                    diesel++;
                }
            }
        }

        System.out.printf("MUITO OBRIGADO\nAlcool: %d\nGasolina: %d\nDiesel: %d\n", alcool, gasolina, diesel);
    }
}
